crates/bevy_time/src/lib.rs

```diff
@@ -68,7 +72,14 @@ impl Plugin for TimePlugin {
 
         app.add_systems(
             First,
-            time_system
+            (time_system, queue_delayed_commands::<()>)
+                .chain()
+                .in_set(TimeSystem)
+                .ambiguous_with(event_update_system),
+        )
+        .add_systems(
+            FixedFirst,
+            queue_delayed_commands::<Fixed>
                 .in_set(TimeSystem)
                 .ambiguous_with(event_update_system),
         )
```

Comment:
I'm not sure this is actually correct, as `Time<()>` seems to be contextual time. Would like to hear what someone knowledgeable on time APIs thinks.

Reply:
My understanding is that for `TimedCommands<()>` we need to check if we're in frame time or fixed time, and place into the correct queues for those.

Reply:
I _think_ the solution here might be to duplicate the way `Time` works. That is, set `TimedCommandQueues`'s `T` parameter to default to `()`, and replace it with the contextually appropriate `TimedCommandQueues`, likely using the same systems that update `Time` based on context too. That _should_ resolve this issue.